### PR TITLE
Fix Golint warning Issue

### PR DESCRIPTION
**Pull request by suramrit**
## What I have done: 
- Added small descriptions to the exported functions that were throwing the golint errors
- Did not change any function definitions or implementations 

## Could be better:
 - Can add to the exported types and function definitions to make them more descriptive

## Impact: 

Goreport for golint improved from 50% to 90%. https://goreportcard.com/report/github.com/suramrit/paul
The remaining warnings are around function definitions which I have not changed

### CODE REVIEW

**Comment by suramrit** [comment]
hmmm.. 😕 Not really sure why the CICD test is failing for essentially not much change to the code base.  @Spazzy757 and contributors any thoughts?

File: pkg/github/webhooks.go [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-	"github.com/Spazzy757/paul/pkg/helpers"
-	"github.com/google/go-github/v32/github"
 	"log"
 	"net/http"
+
+	"github.com/Spazzy757/paul/pkg/helpers"
+	"github.com/google/go-github/v32/github"
 )
 
+//IncomingWebhook handles an imcoing webhook request
```
Comment (Spazzy757) [inline pkg/github/webhooks.go:11]: Typo on incoming

**Comment by Spazzy757** [comment]
Yes this is an issue with uploading coverage report as it is running from a fork. Thank you so much for the contribution! I will finish reviewing it and if it's good I will merge it!

**Review by Spazzy757 — CHANGES_REQUESTED** [review]
Small typo but otherwise good to go

**Comment by suramrit** [comment]
That explains it! Thanks. 

Can you please add "hacktoberfest-accepted" label to the PR. It would count towards my hacktoberfest submissions 🙌

File: pkg/github/webhooks.go (hunk 1 above)
Comment (suramrit) [reply pkg/github/webhooks.go]: Fixed in the new push

**Review by Spazzy757 — APPROVED** [review]

**Pull request merged**